### DISCUSSION TITLE
STROBE Light After V.5

### DISCUSSION THREAD

**Discussion by welyy55** [flybywiresim/aircraft#2670] [Help] [answered]
I wish everyone happy times ..

I have a question .

It was before  a32NX V.5  The STROBE  lighting was strong and high flash

As for the  A32NX V.5  , I might see it as very weak .. Do you agree with me?

**Comment by hotshotp** [comment, marked as answer]
its made so it doesnt leak into the cockpit which was the case before - however it is still realistic
